lab.java

```diff
@@ -12,7 +12,7 @@ public static void main(String[] args) {
         String wordToSearchFor = inputs[1];
         String filename = inputs[2];
         //test
```

Comment:
inputs[0] is never used?

Reply:
Yes your are right, if this would be a proper program I would change it but now I don't care really..